WooCommerce/WooCommerceTests/ViewRelated/SwiftUI/BoldableTextViewTests.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+import XCTest
+@testable import WooCommerce
+
+final class BoldableTextViewTests: XCTestCase {
```

Comment:
Just a thought: testing views are usually quite tricky. I guess that the main intention of the test here is to see if correct areas of provided text are styled as bold and for that we don't necessarily need to have a view. What do you think about extracting the code from `parseBoldableElements` to a separate object and test it?
And the role of the view would be the same as now, to combine all the Elements( or text fragments ) together applying provided style.

Reply:
Yea I agree on testing the logic rather than views directly (I felt the tests on `BoldableTextView` quite hacky). I moved the text parsing logic to a separate struct `BoldableTextParser` in https://github.com/woocommerce/woocommerce-ios/pull/3285/commits/78d1ea47625e0becf31e7242d76036c35516120d